apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/nodes/ReferrerNode/OtherReferrer/OtherReferrer.tsx

```diff
@@ -1,21 +1,15 @@
 import Box from '@mui/material/Box'
-import Divider from '@mui/material/Divider'
-import Popover from '@mui/material/Popover'
 import Stack from '@mui/material/Stack'
 import { BaseReferrer } from '../BaseReferrer'
 
 import { GetJourneyAnalytics_journeyReferrer as JourneyReferrer } from '@core/journeys/ui/useJourneyAnalyticsQuery/__generated__/GetJourneyAnalytics'
-import { MouseEvent, useState } from 'react'
+import ChevronDown from '@core/shared/ui/icons/ChevronDown'
+import { Accordion, AccordionDetails, AccordionSummary } from '@mui/material'
```

Comment:
I think we had a issue with imports importing everything from a module instead of just the functions that was being used, so we've been splitting up the imports
```suggestion
import { Accordion } from '@mui/material/Accordion'
```